.changeset/thirty-knives-begin.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
It's an internal change, we don't need to publish it. Storybook is used for local testing only :)

Reply:
Thanks for clarifying. I removed the changeset :)